Dungeoneer/src/com/interrupt/dungeoneer/entities/Monster.java

```diff
@@ -289,19 +289,22 @@
 	public float spawnMomentumTransfer = 1.0f;
 
 	private transient Entity attackTarget = null;
+    /** Sound played while monster is idle. */
+    @EditorProperty
+    private Float viewDistance = 17.0f;
 
 	public Monster() {
 		// for ranged monsters
 		maxMp = 10;
 		mp = maxMp;
 
 		stepHeight = 0.4f;
-		
+
 		attacktimer = 30;
 		rangedAttackTimer = 30;
-		
-		mass = 2f;
 
+		mass = 2f;
+        viewDistance = 17.0f;
```

Comment:
I think you need to initialize this property also in the second constructor?

Reply:
Because the default value is `17.0f` I don't think it's necessary to set it in the constructor?

Reply:
My bad. That should not be necessary, indeed.